kotlin/src/main/kotlin/io/logto/sdk/core/exception/LogtoException.kt

```diff
@@ -0,0 +1,24 @@
+package io.logto.sdk.core.exception
+
+open class LogtoException(
+    message: String,
+    cause: Throwable? = null,
+) : RuntimeException(message, cause) {
+    class VerificationException(
+        message: Enum<Verification>,
+        cause: Throwable? = null,
+    ) : LogtoException(message.name, cause)
+
+    enum class Verification {
```

Comment:
Similarly here, should it be sth like CallbackUrlVerification? Verification seems should also cover i.e idToken verification errors.

Reply:
updated